package.json

```diff
@@ -29,11 +29,11 @@
     "@astrojs/react": "3.3.0",
     "@emotion/cache": "11.14.0",
     "@emotion/css": "11.13.5",
-    "@emotion/react": "11.14.0",
+    "@emotion/react": "^11.14.0",
     "@emotion/server": "11.11.0",
-    "@emotion/styled": "11.14.0",
+    "@emotion/styled": "^11.14.0",
     "@mui/icons-material": "7.1.0",
-    "@mui/material": "7.1.0",
+    "@mui/material": "^7.1.1",
```

Comment:
You can not bump the version of a functional implementation This kind of action can bring many errors sin the functionality the The material version to 7.1.0